spec/controllers/memes_controller_spec.rb

```diff
@@ -34,4 +34,30 @@
       expect(response_json[1]["user_id"]).to eq user_member.id
     end
   end
-end
\ No newline at end of file
+
+  describe "GET#show" do
+    it "returns successful response with json-formatted data" do
+      get :show, params: {id: meme_1.id}
+
+      expect(response.status).to eq 200
+      expect(response.content_type).to eq "application/json"
+    end
+
+    it "returns the correct meme data in a usable format" do
+      get :show, params: {id: meme_1.id}
+      response_json = JSON.parse(response.body)
+
+      expect(response.status).to eq 200
```

Comment:
These two lines (50 and 51) should be removed, as they're redundant with 42 and 43